mta/mta_services_test.go

```diff
@@ -96,6 +110,18 @@ var _ = Describe("MtaServices", func() {
 			Ω(CopyFile(sourceFilePath, targetFilePath, os.Create)).Should(HaveOccurred())
 		})
 
+		It("Copy file creates the destination folder", func() {
+			jsonData, err := json.Marshal(getMtaInput())
+			Ω(err).Should(Succeed())
+			sourceFilePath := getTestPath("result", "temp.mta.yaml")
+			targetFilePath := getTestPath("result2", "temp2.mta.yaml")
+			Ω(CreateMta(sourceFilePath, string(jsonData), os.MkdirAll)).Should(Succeed())
+			Ω(CopyFile(sourceFilePath, targetFilePath, os.Create)).Should(Succeed())
+			Ω(targetFilePath).Should(BeAnExistingFile())
+			err2 := os.RemoveAll(getTestPath("result2"))
+			Ω(err2).Should(Succeed())
```

Comment:
Instead of deleting it in the test add the deletion to the `AfterEach` that deletes `result`